haiku-apps/colors/colors-2.3.recipe

```diff
@@ -23,8 +26,7 @@ PROVIDES="
 	colors$secondaryArchSuffix = $portVersion
 	app:Colors = $portVersion
 	"
-REQUIRES="
-	"
+
 BUILD_PREREQUIRES="
 	haiku${secondaryArchSuffix}_devel
```

Comment:
It would be better to move the first line of BUILD_PREREQUIRES to BUILD_REQUIRES (instead of removing tha latter completely):
```
REQUIRES="
	haiku${secondaryArchSuffix}_devel"
	"
BUILD_PREREQUIRES="
	makefile_engine
	cmd:gcc$secondaryArchSuffix
	cmd:make
	"
```